### ISSUE TITLE
[SCREENREADER]: Multi-step forms are setting focus incorrectly when users press Back or Continue

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#252]
## Issue
Our multi-step forms usually set focus on the progress bar at the top of each page when users press the `Back` or `Continue` buttons. This serves to position users to enter data quickly, and to notify screen readers they should announce the progress bar heading. Tonight I noticed a regression where the focus is not being moved to the progress bar on route change. Screenshot attached below.

/cc @goldenmeanie @emilywaggoner 

## Acceptance Criteria
* As a keyboard user I want focus to be returned to the progress bar every time I press `Back` or `Continue` on multi-step forms
* As a screen reader user, I want to hear the progress bar announced when new pages load after I press `Back` or `Continue`

## Related Issues
* https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18409

## Environment
* Windows 10
* IE11, Chrome
* JAWS screen reader
* (This behavior can be independently confirmed in NVDA, VoiceOver as needed)
* https://staging.va.gov/health-care/apply/application/introduction
* https://staging.va.gov/disability/file-disability-claim-form-21-526ez/introduction
* Test user vets.gov.user+264

## Steps to Recreate
1. Enter `https://staging.va.gov/health-care/apply/application/introduction` in browser
2. Sign in with user 264
3. Start or continue the Healthcare form, 526 v2 or other multi-step form
4. Tab through the form and press the `Continue` button
5. Visually verify the focus is still on the button on next route load
6. Open Chrome dev tools > console
7. Press `Continue` again, and run `$document.activeElement` to programmatically verify this

## Possible Fixes (optional)
We should return to the previous practice of setting focus on the progress bar. Right now it has a tabindex of 0, this might be okay, or it might need to be changed to -1 to allow programmatic focus with Javascript.

We should also do manual testing with the big three screen readers to ensure full function is restored.

## WCAG or Vendor Guidance (optional)
* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
* [Meaningful Sequence: Understanding SC 1.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

---
![Screen Shot 2019-06-11 at 5 28 17 PM](https://user-images.githubusercontent.com/934879/59311759-49915d80-8c70-11e9-8d30-4a5b9225be3f.png)

---
![Screen Shot 2019-06-11 at 5 28 22 PM](https://user-images.githubusercontent.com/934879/59311770-4eeea800-8c70-11e9-96d2-3dd8455f2f27.png)

**Comment by jbalboni** [comment]
@1Copenut This should be on staging if you wanted to verify it there. Otherwise, I think we can close this.

**Comment by 1Copenut** [comment]
@jbalboni I spot checked the HCA form (logged in and logged out) and the 526 v2 form using Chrome + MacOS. I verified through manual testing the focus was being set properly. I also confirmed it programmatically using `document.activeElement` in the dev tools console. I have high confidence this fixes the issue. Moving to close.

/cc @goldenmeanie
